src/isomorphic/classic/types/ReactPropTypes.js

```diff
@@ -85,6 +85,23 @@ var ReactPropTypes = {
   shape: createShapeTypeChecker,
 };
 
+/**
+ * inlined Object.is polyfill to avoid requiring consumers ship their own
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is
+ */
+/*eslint-disable no-self-compare*/
+function is(x, y) {
```

Comment:
This is intentional and requested by me due to conversations had with @sebmarkbage a couple months ago, don't worry about it. We can deal with figuring out the polyfill story later.